### PR TITLE
[spaceship] Show instructions on how to solve common TestFlight error

### PR DESCRIPTION
**Pull request by KrauseFx**
Fixes https://github.com/fastlane/fastlane/issues/1873
Fixes https://github.com/fastlane/fastlane/issues/4002
and more

### CODE REVIEW

File: spaceship/lib/spaceship/tunes/build_train.rb [hunk 1]
```diff
@@ -141,7 +141,28 @@ def update_testing_status!(new_value, testing_type, build = nil)
           end
         end
 
-        result = client.update_build_trains!(application.apple_id, testing_type, data)
+        begin
+          result = client.update_build_trains!(application.apple_id, testing_type, data)
+        rescue Spaceship::TunesClient::ITunesConnectError => ex
+          if ex.to_s.include?("You must provide an answer for this question")
```
Comment (mfurtak) [inline spaceship/lib/spaceship/tunes/build_train.rb]: Is it possible that this text matches other "questions" than the encryption question? Is there any text more specific to the `ITSAppUsesNonExemptEncryption` key we can match on instead?

**Review by mfurtak — APPROVED** [review]
This seems good, but the text that we match on is very broad. I'd feel better if we could have more specific error text to match on, but if not, :ship: it!

**Comment by KrauseFx** [comment]
@mfurtak Taking a look at this now, great feedback 👍

**Comment by KrauseFx** [comment]
It took a while, and I can confirm that the error message doesn't contain more information about what value is missing.

**Pull request merged**